collections/misc/commentlist.php

```diff
@@ -114,7 +114,52 @@
 		?>
 		<!-- This is inner text! -->
 		<div id="innertext">
-			<h1><?php echo $collMeta['name']; ?></h1>
+			<div class="justify-center">
+				<h1><?php echo $collMeta['name']; ?></h1>
+				<section class="fieldset-like" style="float:right;width:350px;margin:10px;">
```

Comment:
Does this need to be inline styling? If not, let's move it to base.css

Reply:
Sure, I'll create a class! 
P.S. I didn't add this style, it was already there; simply thought it was too specific for a separate class.

Reply:
Ah, my bad. Hard to tell with the addition of the div what is new code and what isn't. But thank you for adding it anyway!